apps/pollbook/barcode-scanner-daemon/src/unitech_ts100_daemon.rs

```diff
@@ -0,0 +1,270 @@
+use color_eyre::eyre::Context;
+use parse_aamva::AamvaDocument;
+use serde_json;
+use serialport::{DataBits, FlowControl, Parity, StopBits};
+use std::io::Write;
+use std::io::{BufRead, BufReader, ErrorKind};
+use std::os::unix::fs::PermissionsExt;
+use std::os::unix::net::{UnixListener, UnixStream};
+use std::process::exit;
+use std::str::from_utf8;
+use std::sync::atomic::{AtomicBool, Ordering};
+use std::sync::{Arc, Mutex};
+use std::thread::sleep;
+use std::time::{Duration, Instant};
+use std::{fs, thread};
+
+use vx_logging::{log, set_source, Disposition, EventId, EventType, Source};
+
+const UDS_PATH: &str = "/tmp/barcodescannerd.sock";
+
+const SOURCE: Source = Source::VxMarkScanControllerDaemon;
+
+// The string denoting the start of an AAMVA-encoded document.
+// 3 ASCII chars '@', 'line feed', 'record separator' expected per "D.12.3 Header" AAMVA 2020 p.50
+const COMPLIANCE_INDICATOR: &[u8] = b"@\n\x1E";
+
+const HEARTBEAT_LOG_INTERVAL: Duration = Duration::from_secs(60);
+const PORT_NAME: &str = "/dev/ttyACM0";
+const BAUD_RATE: u32 = 115200;
+
+use rusb::{DeviceHandle, UsbContext};
+
+mod aamva_jurisdictions;
+mod parse_aamva;
+
+fn reset_scanner() -> rusb::Result<()> {
+    let ctx = rusb::Context::new()?;
+    for device in ctx.devices()?.iter() {
+        let desc = device.device_descriptor()?;
+        if desc.vendor_id() == 0x2745 && desc.product_id() == 0x300a {
+            let mut handle: DeviceHandle<_> = device.open()?;
```

Comment:
Unnecessary type annotation? 
```suggestion
            let mut handle = device.open()?;
```